app/controllers/authentication.js

```diff
@@ -115,167 +116,188 @@ var Controller = function(){
       next();
     }
   }
+
+  
   /*
    |--------------------------------------------------------------------------
    | Login with GitHub
    |--------------------------------------------------------------------------
    */
+
   this.github = function(req, res) {
     var accessTokenUrl = 'https://github.com/login/oauth/access_token';
     var userApiUrl = 'https://api.github.com/user';
-    
+    var teamAdminUrl = 'https://api.github.com/teams/2178412/memberships/';
```

Comment:
you doesn't need to specify team url like this (not easy to configure because of id..). Rather use teams url:
`var userTeamsUrl   = 'https://api.github.com/user/teams';`
and check if user belong to admin team which is pre-defined in configuration file (`nconf..`)

something like this:
```
request.get({url: userTeamsUrl, qs: accessToken, headers: headers, json: true }, function(err, response, teams) {
   ...
   var isAdmin = _.find(response.body, {name: pre_defined_admin_team});
  ...
```